webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -449,7 +450,26 @@ public function viewAction(Request $request, int $submitId)
                 ->getQuery()
                 ->getResult();
 
+            $judgingRunTestcaseIdsInOrder = $this->em->createQueryBuilder()
+                ->from(JudgingRun::class, 'jr')
+                ->select('jr.testcaseid')
+                ->andWhere('jr.judging = :judging')
+                ->setParameter(':judging', $selectedJudging)
+                ->orderBy('jr.endtime')
```

Comment:
If we ever allow running testcases on different machines at the same time (like @ubergeek42 was working on) this can break right?

Reply:
fixed by only displaying the message when the judging is done